### ISSUE TITLE
Confusing blog content

### ISSUE THREAD

**Issue by dpshelio** [Gaurav17Joshi/Gaurav17Joshi.github.io#1]
@Gaurav17Joshi, it seems this repository is the same than that from @Jash-Shah (https://github.com/Jash-Shah/Blogs). Or at least the post content is the same. This has produced some kind of problems on the OA aggregator page as Jash appears when he shouldn't.  Also, if I go to your blog page, but without "Blog" in the url this is the content that appears.

I'd suggest to remove the posts, or create a new blank page that links to your blog.

**Comment by Gaurav17Joshi** [comment]
I used his template to make my blog, so that my be creating the issue, I will make a new one.

**Comment by dpshelio** [comment]
Yes, we are using the correct link, but the first post, as you said, was with Jash name. I've fixed it on the OA site. The problem I faced was that when I removed the Blog part from the url it shows almost the same page, but with entries that are from Josh, and therefore very confusing for your future visitors. That's why I suggested to remove the content from this repository.
